addons/website_sale/views/templates.xml

```diff
@@ -209,8 +213,22 @@
             <div class="o_wsale_product_information position-relative d-flex flex-column flex-grow-1 flex-shrink-1">
                 <div class="o_wsale_product_information_text flex-grow-1">
                     <h6 class="o_wsale_products_item_title mb-2">
-                        <a class="text-primary text-decoration-none" itemprop="name" t-att-href="product_href" t-att-content="product.name" t-field="product.name" />
-                        <a t-if="not product.website_published" role="button" t-att-href="product_href" class="btn btn-sm btn-danger" title="This product is unpublished.">
+                        <a
+                            class="text-primary text-decoration-none"
+                            itemprop="name"
+                            t-att-href="compute_product_url(product)"
+                            t-nocache="The product URL must stay up to date."
+                            t-att-content="product.name"
+                            t-field="product.name"
+                        />
+                        <a
+                            t-if="not product.website_published"
+                            role="button"
+                            t-att-href="compute_product_url(product)"
+                            t-nocache="The product URL must stay up to date."
+                            class="btn btn-sm btn-danger"
+                            title="This product is unpublished."
+                        >
```

Comment:
1) there is no cache here anymore (the t-nocache is not necessary anymore I believe)
2) why don't you compute the url once (like before) ?

Reply:
Hi @Feyensv 

I think this part is my doing so I'll answer your comment :sweat_smile: At the time this was developed, the PR was not targeting master, so the `t-nocache` was needed and we needed to compute the URL each time as `t-set` are always cached.

I don't think this part is needed if there is no cache (even in the parent).